src/carma-1-tenth/plot_crosstrack_error.py

```diff
@@ -97,6 +98,8 @@ def plot_absolute_route_deviation(bag_dir, start_offset=0.0):
 
     print("Average Deviation:", np.mean(np.abs(route_deviations)))
     print("Maximum Deviation:", np.max(np.abs(route_deviations)))
+    # plt.plot(route_x_points, route_y_points)
```

Comment:
If these are left commented out we should identify what they do with another comment.

Reply:
Those are no longer needed, I removed them in my most recent commit